src/app/logic/calculatePoints.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/extensions
+import { DIFFICULTY_LEVELS } from '../data/consts.js';
+
+export function calculatePoints(
+  correctAnswersScore,
+  isAnswerCorrect,
+  difficultyLevelsProperty,
+) {
+  if (
+    DIFFICULTY_LEVELS[difficultyLevelsProperty].calculatingPoints ===
+    DIFFICULTY_LEVELS.medium.calculatingPoints
```

Comment:
This looks a bit overcomplicated, could you try to think of a way to simplify it?

Reply:
Changed! with isMediumMode bool